dvc/remote/gdrive.py

```diff
@@ -272,12 +282,27 @@ def get_remote_item(self, name, parents_ids):
 
         query += " and trashed=false and title='{}'".format(name)
 
+        param = {
+            "q": query,
+            # Remote might contain items with duplicated titles
+            "maxResults": 1,
+            "corpora": self.corpora,
+        }
+
+        if self.remote_drive_id:
+            param["driveId"] = self.remote_drive_id
+
         # Limit found remote items count to 1 in response
-        item_list = self.drive.ListFile(
-            {"q": query, "maxResults": 1}
-        ).GetList()
+        item_list = self.drive.ListFile(param).GetList()
         return next(iter(item_list), None)
 
+    @gdrive_retry
+    def get_remote_drive_id(self, remote_id):
+        param = {"id": remote_id}
+        item = self.drive.CreateFile(param)
```

Comment:
can we use something less confusing - GetFile? Something that would not create file if it's missing for some reason (wrong remote location)? I would really try to avoid side effects here

Reply:
`PyDrive.file.CreateFile` doesn't create remote file unless `Upload` is called (not called in this case). We might need to rename that PyDrive's API to avoid such confusion.

Reply:
yeah, I remember that it's not the first time I ask :) could you small comment around please?